lib/msal-angular/src/msal.service.ts

```diff
@@ -304,11 +304,11 @@ export class MsalService extends UserAgentApplication {
         return new Promise((resolve, reject) => {
             super.acquireTokenSilent(scopes, authority, user, extraQueryParameters).then((token: any) => {
                 this._renewActive = false;
-                this.broadcastService.broadcast('msal:acquireTokenSuccess', token);
+                this.broadcastService.broadcast('msal:acquireTokenSuccess', {token});
                 resolve(token);
             }, (error: any) => {
                 this._renewActive = false;
-                this.broadcastService.broadcast('msal:acquireTokenFailure', error);
+                this.broadcastService.broadcast('msal:acquireTokenFailure', {error});
```

Comment:
I think we should pass both error and error_description to all the failure events as both these fields are always returned from the server in the case of a failed response. I believe that instead of a dynamic object, it will be better to pass a named object with these fields (token,token_type,error, error_description &scopes) in all broadcast events and setting the relevant fields to null for specific event. For instance, set error and error_desc to null in a successful response and set token and token_type to null in a failed response. This will reduce the error of forgetting to add a required field to the broadcast event object by mistake.